src/core/util/errorMessageFormatter.js

```diff
@@ -1,7 +1,21 @@
 const { ValidationError } = require('ajv')
+/**
+ * Module Error Message Formatter.
+ * 
+ * @module errorMessageFormatter
+ */
 
+/**
+ * ErrorMessageFormatter class.
```

Comment:
This class comment is not consistent to the others (they are describing much more). Is this intended (because this class name is very self-explaining?)

Reply:
Updated